Networking/NetworkingTests/Remote/ProductReviewsRemoteTests.swift

```diff
@@ -34,78 +34,81 @@ final class ProductReviewsRemoteTests: XCTestCase {
 
     /// Verifies that loadAllProductReviews properly parses the `reviews-all` sample response.
     ///
-    func testLoadAllProductReviewsProperlyReturnsParsedProductReviews() {
+    func testLoadAllProductReviewsProperlyReturnsParsedProductReviews() throws {
         // Given
         let remote = ProductReviewsRemote(network: network)
         let expectation = self.expectation(description: "Load All Product Reviews")
 
         network.simulateResponse(requestUrlSuffix: "products/reviews", filename: "reviews-all")
 
         // When
-        var result: (reviews: [ProductReview]?, error: Error?)
-        remote.loadAllProductReviews(for: sampleSiteID) { reviews, error in
-            result = (reviews, error)
+        var result: Result<[ProductReview], Error>?
+        remote.loadAllProductReviews(for: sampleSiteID) { loadResult in
+            result = loadResult
             expectation.fulfill()
         }
```

Comment:
nit: can save some unwrapping (similar for the other test cases in the diffs)

```suggestion
        let result: Result<[ProductReview], Error> = try waitFor { promise in
            remote.loadAllProductReviews(for: sampleSiteID) { result in
                promise(result)
            }
        }
```

Reply:
Updated in 75ae195675812bd7e3454ae8db51e041dc08e8a8 👍